server/sonar-server/src/main/java/org/sonar/server/computation/container/ComputeEngineContainerImpl.java

```diff
@@ -34,6 +34,7 @@
 import org.sonar.api.config.PropertyDefinitions;
 import org.sonar.api.utils.log.Logger;
 import org.sonar.api.utils.log.Loggers;
+import org.sonar.ce.settings.ThreadLocalSettings;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.sonar.ce.settings.ThreadLocalSettings'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUselessImportCheck)